agent/agent.go

```diff
@@ -1242,6 +1246,27 @@ func (a *Agent) killProcs(ctx context.Context, tc *taskContext, ignoreTaskGroupC
 	}
 }
 
+func (a *Agent) clearGitConfig(tc *taskContext, reason string) {
+	logger := grip.NewJournaler("clearGitConfig")
```

Comment:
Grip is already set up for the agent with a default logger that goes to a file, and it would be preferable if we used that instead of making a brand new logger with different configuration. It can be retrieved with `grip.GetDefaultJournaler`.